mappings/net/minecraft/enchantment/Enchantment.mapping

```diff
@@ -15,44 +15,63 @@ CLASS net/minecraft/unmapped/C_jxtrubuh net/minecraft/enchantment/Enchantment
 		ARG 0 id
 	METHOD m_fecfwekl getRarity ()Lnet/minecraft/unmapped/C_jxtrubuh$C_dnkiztld;
 	METHOD m_glfcpyqa getMaxLevel ()I
+		COMMENT Sets the maximum number of levels the enchantment has. Default: 1.
```

Comment:
What's meant by levels here? XP?

Reply:
@NoComment1105 clearing this up will make this pr ready to merge. I want to get as much in before 1.18 is released.

Reply:
Feel free to merge without, it was more of a curiosity than a concern

Reply:
Yes Levels of XP sorry i missed this i'll change it now

Reply:
wait not levels of xp i'm dumb, tiers as in Sharpness 1 Sharpness 5